.github/scripts/create-platform-release-pr.sh

```diff
@@ -404,6 +404,14 @@ Platform: ${platform}"
         echo "Version bump committed"
     fi
 
+    # If the version bump branch has no commits ahead of main, skip pushing/PR creation
+    # right-only count gives number of commits unique to the version bump branch
+    ahead_count=$(git rev-list --right-only --count "${main_branch}...${version_bump_branch_name}" || echo 0)
```

Comment:
Why do we fallback to `echo 0` here?
If `git rev-list` fails (e.g. due to a bad ref), this would set `ahead_count=0` and the script would continue.
It seems to me that could result in hiding unexpected errors? (which I don't think we should)